### PR TITLE
Common Lisp: enable fuzzy matching with comapny

### PR DESCRIPTION
**Pull request by tuhdo**

### CODE REVIEW

**Comment by TheBB** [comment]
Dude, link please.

https://github.com/syl20bnr/spacemacs/issues/7019

**Comment by tuhdo** [comment]
Well that issue is different. It addresses fuzzy matching with `capf` in general, while this package uses the specialized fuzzy matcher from SLIME (not `capf`).

**Comment by TheBB** [comment]
Okay, sorry.

File: layers/+lang/common-lisp/packages.el [hunk 1]
```diff
@@ -33,6 +34,10 @@
 (defun common-lisp/post-init-helm-gtags ()
   (spacemacs/helm-gtags-define-keys-for-mode 'common-lisp-mode))
 
+(defun common-lisp/init-slime-company ()
+  :config
```
Comment (TheBB) [inline layers/+lang/common-lisp/packages.el]: ?

Comment (tuhdo) [inline layers/+lang/common-lisp/packages.el]: I use `:config` to delay setting of the variable until the package is loaded. That is, when the variable is available.

Comment (TheBB) [inline layers/+lang/common-lisp/packages.el]: Yeah, but it goes in `use-package`. Defun doesn't understand it.

Comment (TheBB) [inline layers/+lang/common-lisp/packages.el]: Besides there's no need to delay setting variables until the defvar has happened, for that matter.

Comment (tuhdo) [inline layers/+lang/common-lisp/packages.el]: Oh it's a typo. I deleted the line `use-package` somehow.

Comment (tuhdo) [inline layers/+lang/common-lisp/packages.el]: Actually I deleted `use-package`, but left `:config` behind. I've updated.

**Comment by tuhdo** [comment]
I updated the PR.

**Comment by AndreaOrru** [comment]
Is this getting merged?

**Comment by TheBB** [comment]
Thanks! Cherry-picked in develop. You can safely delete your branch.

**Pull request closed without merging**